x/ibc/07-tendermint/client_state.go

```diff
@@ -21,18 +23,52 @@ var _ clientexported.ClientState = ClientState{}
 type ClientState struct {
 	// Client ID
 	ID string `json:"id" yaml:"id"`
+	// Duration of the period since the LastestTimestamp during which the
+	// submitted headers are valid for upgrade
+	TrustingPeriod time.Duration `json:"trusting_period" yaml:"trusting_period"`
```

Comment:
Do we need both trusting and unbonding? Can't we default trusting to a fraction of the unbonding period?